web/src/main/webapp/WEB-INF/pages/application_state.jsp

```diff
@@ -106,24 +134,90 @@
+                              <c:when test="${data.recommendation == 0}">
+                                <spring:message code="common.no"/>
+                              </c:when>
+                            </c:choose>
+                          </p>
+                          <c:if test="${stat.first}">
+                            <select required class="stateInput" id="recommendationInput" style="display:none;"
+                              data-bind="valueWithInit: 'recommendation'">
+                              <option disabled ${data.recommendation == null ? 'selected' : ''}><spring:message code="common.pleaseChoose"/></option>
+                              <option value="1" ${data.recommendation == 1 ? 'selected' : ''}><spring:message code="common.yes"/></option>
+                              <option value="0" ${data.recommendation == 0 ? 'selected' : ''}><spring:message code="common.no"/></option>
+                            </select>
+                          </c:if>
+                        </div>
+                        <div class="help-block with-errors"></div>
+                      </div>
+                      <div class="form-group">
+                        <label for="reviewerNameInput"  class="control-label col-sm-4"><spring:message code="statehistory.field.reviewerName"/></label>
+                        <div class="col-sm-8">
+                          <p class="form-control-static <c:if test='${stat.first}'>stateData</c:if>" id="reviewerNameP">${data.reviewerName}</p>
+                          <c:if test="${stat.first}">
+                            <input type="text" class="stateInput" id="reviwerNameInput" style="display:none" value="${data.reviewerName}"
+                              data-bind="valueWithInit: 'reviewerName'"
+                              data-error="<spring:message code='statehistory.error.reviewerName.length'/>"
+                              pattern="^.{3,100}$" required />
+                          </c:if>
+                        </div>
+                        <div class="help-block with-errors"></div>
+                      </div>
+                      <div class="form-group">
+                        <label for="recommendedPositionLevelInput"  class="control-label col-sm-4"><spring:message code="statehistory.field.recommendedPositionLevel"/></label>
+                        <div class="col-sm-8">
+                          <p class="form-control-static <c:if test='${stat.first}'>stateData</c:if>" id="reviewerNameP">L${data.recommendedPositionLevel}</p>
+                          <c:if test="${stat.first}">
+                              <select required class="stateInput" id="recommendedPositionLevelInput"
+                                data-bind="valueWithInit: 'recommendedPositionLevel'"
+                                data-error="<spring:message code='statehistory.error.recommendedPositionLevel.range'/>"
+                                style="display:none;">
```

Comment:
As in hte case of the common error messages for state history PR, remove the style attribute and add a hidden class from bootstrap.